gender_novels/common.py

```diff
@@ -10,11 +10,11 @@
 METADATA_LIST = ['gutenberg_id', 'author', 'date', 'title', 'country_publication', 'author_gender',
                  'subject', 'corpus_name', 'notes']
 
-# 30 books from gutenberg downloaded from Dropbox folder shared with Keith,
+# 30 books from gutenberg downloaded from Dropbox folder shared with Keith, except it's no longer actually 30 books
```

Comment:
Could you specify how many books it now contains?

Reply:
Well, there are 62 on my local branch, but only 32 in the origin.  I'm not sure I want to push more text files that won't be of use to anyone else.  Should I?